src/vl-mapactions-snap-interaction.js

```diff
@@ -1,13 +1,14 @@
 import Snap from 'ol/interaction/Snap';
 
 export class VlSnapInteraction extends Snap {
-  constructor(layer) {
-    const source = layer ? layer.getSource() : null;
-    const snapOptions = {
-      source: source,
-      pixelTolerance: 7,
-    };
+  constructor(source, options) {
+    const snapOptions = Object.assign({}, options);
+    snapOptions.source = source;
+    if (options && options.pixelTolerance) {
+      snapOptions.pixelTolerance = options.pixelTolerance;
+    } else {
+      snapOptions.pixelTolerance = 7;
+    }
     super(snapOptions);
```

Comment:
```suggestion
  constructor(source, options = {}) {
    const snapOptions = Object.assign({}, options);
    snapOptions.source = source;
    snapOptions.pixelTolerance = options.pixelTolerance || 7;
    super(snapOptions);
  }
```

Alleen Elvis blijft bestaan en maakt het (voor mij) iets leesbaarder.